src/charm.py

```diff
@@ -95,7 +95,8 @@ def __init__(self, *args: Any) -> None:
         self._prometheus_scrape_relation_name = "metrics-endpoint"
         self._loki_push_api_relation_name = "logging"
         self._kratos_service_command = "kratos serve all"
-        self._log_path = "/var/log/kratos.log"
+        self._log_dir = Path("/var/log")
```

Comment:
Do we need to make it a `Path` object if it needs to be passed as a string afterwards?